test/testApolloServer.js

```diff
@@ -185,6 +200,33 @@ describe('ApolloServer', () => {
       );
     });
   });
+
+  it('will append resolvers to a js schema', () => {
+    const app = express();
+    const jsServer = apolloServer({
+      schema: testJSSchema,
+      resolvers: testResolvers,
+      mocks: {
+        RootQuery: () => ({
+          stuff: () => 'stuffs',
+          useTestConnector: () => 'utc',
+          species: 'rawr',
+        }),
+      }
+    });
+    app.use('/graphql', jsServer);
+    const expected = {
+      stuff: 'stuffs',
+      useTestConnector: 'utc',
+      species: 'rawr',
+    };
+    return request(app).get(
+      '/graphql?query={stuff useTestConnector species(name: "uhu")}'
+    ).then((res) => {
+      return expect(res.body.data).to.deep.equal(expected);
```

Comment:
I think this doesn't actually do what one would expect, because the mocks interfere with resolvers. There's another PR open for that, but until that's merged, this test will actually use the mocks, and not the resolvers (the coveralls coverage report also seems to indicate that the new added line is never executed).